2024/minutes/RISC-V 标准 PLIC 与平头哥玄铁 PLIC.md

```diff
@@ -0,0 +1,261 @@
+| 0x0002300 | PLIC_H3_MIE0   | R/W  | 0x0    | 核 3 1 ～31 号机器模式中断使能寄存器         |
+| 0x0002304 | PLIC_H3_MIE1   | R/W  | 0x0    | 核 3 32 ～63 号机器模式中断使能寄存器        |
+| …         | …              | …    | …      | …                                            |
+| 0x000237C | PLIC_H3_MIE31  | R/W  | 0x0    | 核 3 992 ～1023 号机器模式中断使能寄存器     |
+| 0x0002380 | PLIC_H3_SIE0   | R/W  | 0x0    | 核 3 1 ～31 号超级用户模式中断使能寄存器     |
+| 0x0002384 | PLIC_H3_SIE1   | R/W  | 0x0    | 核 3 32 ～63 号超级用户模式中断使能寄存器    |
+| …         | …              | …    | …      | …                                            |
+| 0x00023FC | PLIC_H3_SIE31  | R/W  | 0x0    | 核 3 992 ～1023 号超级用户模式中断使能寄存器 |
+| Reserved  | -              | -    | -      | -                                            |
+| 0x01FFFFC | PLIC_CTRL      | R/W  | 0x0    | PLIC 权限控制寄存器                          |
+| 0x0200000 | PLIC_H0_MTH    | R/W  | 0x0    | 核 0 机器模式中断阈值寄存器                  |
+| 0x0200004 | PLIC_H0_MCLAIM | R/W  | 0x0    | 核 0 机器模式中断响应/完成寄存器             |
+| Reserved  | -              | -    | -      | -                                            |
+| 0x0201000 | PLIC_H0_STH    | R/W  | 0x0    | 核 0 超级用户模式中断阈值寄存器              |
+| 0x0201004 | PLIC_H0_SCLAIM | R/W  | 0x0    | 核 0 超级用户模式中断响应/完成寄存器         |
+| Reserved  | -              | -    | -      | -                                            |
+| 0x0202000 | PLIC_H1_MTH    | R/W  | 0x0    | 核 1 机器模式中断阈值寄存器                  |
+| 0x0202004 | PLIC_H1_MCLAIM | R/W  | 0x0    | 核 1 机器模式中断响应/完成寄存器             |
+| Reserved  | -              | -    | -      | -                                            |
+| 0x0203000 | PLIC_H1_STH    | R/W  | 0x0    | 核 1 超级用户模式中断阈值寄存器              |
+| 0x0203004 | PLIC_H1_SCLAIM | R/W  | 0x0    | 核 1 超级用户模式中断响应/完成寄存器         |
+| Reserved  | -              | -    | -      | -                                            |
+| 0x0204000 | PLIC_H2_MTH    | R/W  | 0x0    | 核 2 机器模式中断阈值寄存器                  |
+| 0x0204004 | PLIC_H2_MCLAIM | R/W  | 0x0    | 核 2 机器模式中断响应/完成寄存器             |
+| Reserved  | -              | -    | -      | -                                            |
+| 0x0205000 | PLIC_H2_STH    | R/W  | 0x0    | 核 2 超级用户模式中断阈值寄存器              |
+| 0x0205004 | PLIC_H2_SCLAIM | R/W  | 0x0    | 核 2 超级用户模式中断响应/完成寄存器         |
+| Reserved  | -              | -    | -      | -                                            |
+| 0x0206000 | PLIC_H3_MTH    | R/W  | 0x0    | 核 3 机器模式中断阈值寄存器                  |
+| 0x0206004 | PLIC_H3_MCLAIM | R/W  | 0x0    | 核 3 机器模式中断响应/完成寄存器             |
+| Reserved  | -              | -    | -      | -                                            |
+| 0x0207000 | PLIC_H3_STH    | R/W  | 0x0    | 核 3 超级用户模式中断阈值寄存器              |
+| 0x0207004 | PLIC_H3_SCLAIM | R/W  | 0x0    | 核 3 超级用户模式中断响应/完成寄存器         |
+| Reserved  | -              | -    | -      | -                                            |
+
+
+## 玄铁 C907 & C908 R1S0 & C920 R2S3 PLIC
```

Comment:
There exists PLIC_CTRL register like C906, please re-check.

Reply:
Thank you for pointing that out! You are correct that the PLIC_CTRL register exists in the C906 and other XuanTie C-series chips. I have rechecked the documentation and confirmed that the PLIC_CTRL register is indeed present across the XuanTie C-series, including C906, C920 R1S6, C910, C907, C908 R1S0, and C920 R2S3.

To ensure clarity, I have added a summary of the interrupt-related registers at the end of the document, which includes the PLIC_CTRL register and its functionality.